src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+// Отримуємо всі елементи з класом "number"
+function calculatePopulation() {
+  const populationElements = document.querySelectorAll('.population');
+  const populations = [];
+
+  // 1. Get text data and convert it to numbers
+  populationElements.forEach((el) => {
+    const populationStr = el.textContent.replace(/,/g, ''); // Remove commas
+    const populationNum = parseInt(populationStr, 10);
+
+    if (!isNaN(populationNum)) {
+      populations.push(populationNum);
+    }
+  });
+
+  // 2. Calculate total and average
+  const totalPopulation = populations.reduce((acc, num) => acc + num, 0);
+  const averagePopulation = totalPopulation / populations.length;
```

Comment:
There is a potential division by zero error here if the 'populations' array is empty. Consider adding a check to ensure that the array is not empty before performing the division.